frontend/__test__/SelectTest.test.js

```diff
@@ -0,0 +1,55 @@
+import { renderRouter, screen, fireEvent } from 'expo-router/testing-library';
+import Select from '../app/components/Select';
+
+it('renders the correct title', async () => {
+    const items = [
+        { label: 'Chile', value: 'Chile' },
+        { label: 'Peru', value: 'Peru' },
+    ];
+    const title = 'this is title';
+    const onChange = jest.fn();
+    
+
+    const MockComponent = jest.fn(() => <Select items={items} title={title} onChange={onChange} />);
+
+    renderRouter(
```

Comment:
hvis du flytter det her om i en seperat fil crasher det så? fordi det kunne være nice at gøre det

alternative kunne du have det hele i en describe og så de enkelte tests i "it"s under den describe